Dropdown/Dropdown.js

```diff
@@ -101,13 +158,13 @@ const DropdownBase = kind({
 		'data-spotlight-id': PropTypes.string,
```

Comment:
Is this needed for now?

Reply:
I don't see it to be used. I will remove it